src/include/homestore/btree/mem_btree.hpp

```diff
@@ -69,6 +69,8 @@ class MemBtree : public Btree< K, V > {
         return btree_status_t::success;
     }
 
+    void update_new_root_info(bnodeid_t root_node, uint64_t version) override {}
```

Comment:
Why is this method defined in base class of Btree and not just in IndexTable (derived) class?

Reply:
Any user of the btree class should be informed about the new root created, so that they can store it somewhere. So added it in base class